fbw-a380x/src/systems/instruments/src/MsfsAvionicsCommon/UiWidgets/RadioButtonGroup.tsx

```diff
@@ -78,12 +84,21 @@ export class RadioButtonGroup extends DisplayComponent<RadioButtonGroupProps> {
     this.subs.push(
       this.props.color.sub((v) => {
         this.props.values.forEach((val, idx) => {
-          document.getElementById(`${this.props.idPrefix}_label_${idx}`)?.classList.remove('yellow');
-          document.getElementById(`${this.props.idPrefix}_label_${idx}`)?.classList.remove('green');
-          document.getElementById(`${this.props.idPrefix}_label_${idx}`)?.classList.remove('cyan');
-          document.getElementById(`${this.props.idPrefix}_label_${idx}`)?.classList.remove('white');
-
-          document.getElementById(`${this.props.idPrefix}_label_${idx}`)?.classList.add(v);
+          document
```

Comment:
Can we store a ref to the element rather than fetching it with `getElementById`?